### DISCUSSION TITLE
Activating sleap

### DISCUSSION THREAD

**Discussion by sofiamarinquiros** [talmolab/sleap#1370] [Help!] [answered]
Hello!
So I installed sleap and I was opening up the GUI with no problems and I closed the terminal. When I opened the terminal and tried to activate sleap again I ran into some issues. When I ran conda activate sleap I got an error saying "EnvironmentNameNotFound: Could not find conda environment: sleap
You can list all discoverable environments with `conda info --envs`." 
I then ran conda info -- envs and got 
"# conda environments:
#
                         /Users/sofiamarinquiros/anaconda3
                         /Users/sofiamarinquiros/anaconda3/envs/DEEPLABCUT
                         /Users/sofiamarinquiros/anaconda3/envs/sleap
base                  *  /Users/sofiamarinquiros/mambaforge
                         /Users/sofiamarinquiros/miniconda3"
The sleap environment is clearly there but for some reason is not activating. Anyone have any suggestions?

**Comment by roomrys** [comment]
Hi @sofiamarinquiros,

Can you do a `mamba info --envs`? And if things look as expected, a `mamba activate sleap`?

Let me know if that works!

Thanks,
Liezl

**Comment by talmo** [comment, marked as answer]
FYI @sofiamarinquiros: it looks like you have 3 different Python environment managers (Anaconda, Miniconda and Mambaforge) and that they're all in your `PATH` which can create all sorts of conflicts. I'd strongly recommend just getting rid of all 3 and just installing one (we recommend Mambaforge).

**Reply by sofiamarinquiros** [reply]
Thankyou so much! This solved the issue.
